opendevin/runtime/docker/ssh_box.py

```diff
@@ -280,6 +296,23 @@ def __init__(
         atexit.register(self.close)
         super().__init__()
 
+    def init_plugins(self, requirements: list[PluginRequirement]):
```

Comment:
Since we plan to pre-build using Dockerfile later -- how about we hold off these docker commit implementations (as if `use_cache=False`) and directly go for a better Dockerfile-based solution?

Reply:
Great idea! I will add support for this.